modules/ROOT/pages/constraints/examples.adoc

```diff
@@ -1492,6 +1588,79 @@ Constraint already exists: Constraint( id=24, name='part_of', type='RELATIONSHIP
 ======
 
 
+[discrete]
+[[constraints-create-relationship-property-type-constraints-on-invalid-types]]
+=== Creating constraints on invalid types will fail
+
+
+.+CREATE CONSTRAINT+
+======
+
+Create a relationship property type constraint restricting the property `releaseOrder` to integer values excluding `null` on relationships with the `PART_OF` relationship type:
+
+.Query
+[source, cypher, role=test-fail]
+----
+CREATE CONSTRAINT score
+FOR ()-[part:PART_OF]-() REQUIRE part.releaseOrder IS :: INTEGER NOT NULL
+----
+
+In this case, the constraint cannot be created because excluding `null` is not allowed in property type constraints.
+To also ensure that the property exists (is not `null`), add an existence constraint on the property.
+
+.Error message
+[source, error]
+----
+Failed to create relationship property type constraint: Invalid property type `INTEGER NOT NULL`.
+----
+
+======
+
+.+CREATE CONSTRAINT+
+======
+
+Create a relationship property type constraint restricting the property `releaseOrder` to list of nullable integer values on relationships with the `PART_OF` relationship type:
+
+.Query
+[source, cypher, role=test-fail]
+----
+CREATE CONSTRAINT score
+FOR ()-[part:PART_OF]-() REQUIRE part.releaseOrder IS :: LIST<INTEGER>
+----
+
+In this case, the constraint cannot be created because the inner type of list types cannot be nullable.
```

Comment:
```suggestion
In this case, the constraint cannot be created because the inner type of list types cannot be nullable. The correct type to use for the constraint is `LIST<INTEGER NOT NULL>` because null values cannot be stored as part of a list.
```

Reply:
this has been added as line 1633